cloud/aws/templates/aws_oidc/bin/setup.py

```diff
@@ -1,27 +1,48 @@
-import os
 import subprocess
 import shlex
-import shutil
+from getpass import getpass
+from typing import Dict
 
+from cloud.aws.templates.aws_oidc.bin.aws_cli import AwsCli
 from cloud.aws.templates.aws_oidc.bin.aws_template import AwsSetupTemplate
 from cloud.aws.bin.lib import backend_setup
+from cloud.shared.bin.lib.config_loader import ConfigLoader
+
+# TODO: move these to variable_definitions.json and read docs from there.
+# Map of secrets that need to be set by the user and can't be empty values.
+# Key is the name of the secret without app prefix, value is doc shown to user
+# if the secret is unset.
+SECRETS: Dict[str, str] = {
+    'adfs_client_id':
+        'Client id for the ADFS configuration. Enter any value if you do not use ADFS.',
+    'adfs_secret':
+        'Secret for the ADFS configuration. Enter any value if you do not use ADFS.',
+    'applicant_oidc_client_id':
+        'Client ID for your OIDC provider. Enter any value if you have not set it up yet.',
+    'applicant_oidc_client_secret':
+        'Client secret for your OIDC provider. Enter any value if you have not set it up yet.',
+}
 
 
 class Setup(AwsSetupTemplate):
 
-    def get_current_user(self):
-        get_current_command = "aws sts get-caller-identity --query UserId --output text"
+    def __init__(self, config: ConfigLoader):
+        super().__init__(config)
+        self.aws_cli = AwsCli(config)
```

Comment:
Not blocking: If this shouldn't be accessed outside of the class, I'd make it "private" (e.g. `self._aws_cli`).

Reply:
Done.